docs/content/preview/yugabyte-platform/prepare/cloud-permissions/cloud-permissions-nodes-k8s.md

```diff
@@ -137,7 +137,7 @@ export YBA_NAMESPACE="yb-platform"
 
 curl -s https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-namespaced.yaml \
   | sed "s/namespace: <SA_NAMESPACE>/namespace: ${YBA_NAMESPACE}"/g \
-  | kubectl apply -n ${YBA_NAMESPACE} -f -
+  | kubectl apply -n ${<target_namespace>} -f -
```

Comment:
Is this applicable only for the "Namespace Restricted" and not "Namespace Admin", "Global Restricted", or "Global Admin" ?

Reply:
Yes, this is only applicable to namespace based rbac. 
So, it should be applicable to "Namespace restricted" and "Namespace admin"